src/project.cc

```diff
@@ -35,6 +35,28 @@ void Project::save_files(const boost::filesystem::path &path) {
   }
 }
 
+void Project::on_save() {
+  if(Notebook::get().get_current_page()==-1)
+    return;
+  auto view=Notebook::get().get_current_view();
+  if(view->file_path.filename()=="CMakeLists.txt") {
+    auto build=get_build(view->file_path);
```

Comment:
What about CMake-modules? For instance `.cmake` files. If I edit `FindLibClang` for instance then we might need a full re-parse.